llvm/lib/Target/AArch64/SVEInstrFormats.td

```diff
@@ -8929,15 +8905,15 @@ class sve_fp_matrix_mla<bit sz, string asm, ZPRRegOp zprty>
 
   let Constraints = "$Zda = $_Zda";
   let DestructiveInstType = DestructiveOther;
-  let ElementSize = zprty.ElementSize;
+  let ElementSize = reg_ty.ElementSize;
```

Comment:
```suggestion
  let ElementSize = zda_ty.ElementSize;
```
I believe the `ElementSize` of the `MOVPRFX` should be that of the destination register. That said, I'm not sure it really matters in this case if `FMMLA` can only be followed by an unpredicated `MOVPRFX`?

Reply:
Done